tests/slow/src/test/scala/tests/sbt/SbtSlowSuite.scala

```diff
@@ -245,22 +245,17 @@ object SbtSlowSuite extends BaseImportSuite("sbt-import") {
       )
       _ = assertNoDiff(client.workspaceDiagnostics, "")
       _ = {
-        assertNoDiff(
-          client.workspaceClientCommands,
-          List(
-            ClientCommands.ReloadDoctor.id,
-            ClientCommands.RunDoctor.id
-          ).mkString("\n")
-        )
+        val expected = ClientCommands.ReloadDoctor.id :: ClientCommands.RunDoctor.id :: Nil
+        val actual = client.workspaceClientCommands
+        assert(actual.startsWith(expected))
```

Comment:
What is the motivation for this change? The ScalaTest error message inside asserts for large lists is quite difficult to read. The output format from `assertNoDiff` is easier to debug in my experience (although it may take some time to get used to the output)

Reply:
because now the `client.workspaceClientCommands` contain also ClientCommands.RefreshModel, so the `noDiff` ` actually won't do.